std/stdio.d

```diff
@@ -44,38 +44,38 @@ version (CRuntime_Glibc)
     version = GCC_IO;
     version = HAS_GETDELIM;
 }
-
-version (OSX)
+else version (CRuntime_Bionic)
```

Comment:
Added `else`'s as made sense, and switched Bionic to use `getdelim`, as it was added with Android API 18.